src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx

```diff
@@ -72,6 +72,42 @@ const addValidUser = async () => {
   await waitFor(() => expect(nextButton).toBeEnabled());
 };
 
+const addInvalidUser = async () => {
```

Comment:
The helper functions could be cleaned up a bit.

I think we should have separate functions for clicking the "Add user" button, a separate function for filling in the username and a function for filling in the ssh key. That way we can chain them and easily use in the tests themselves so they'd be a bit more transparent.

So in the test, I'd expect to see something like:
```
goToUserStep()
clickAddUser()
addUsername(username)
addSshKey(sshKey)
```

That way you can also add a string as an argument and reuse the function for both valid and invalid values where needed.

Currently the `addInvalidUser` and `addInvalidSSHKeyUser` overlap quite a bit. Separating the steps would reduce the duplicity.

Reply:
like this approach thank you!